loadtest/loadtest.go

```diff
@@ -0,0 +1,328 @@
+package loadtest
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"time"
+
+	"github.com/giantswarm/apiextensions/pkg/apis/core/v1alpha1"
+	"github.com/giantswarm/apprclient"
+	"github.com/giantswarm/backoff"
+	"github.com/giantswarm/e2e-harness/pkg/framework"
+	"github.com/giantswarm/helmclient"
+	"github.com/giantswarm/microerror"
+	"github.com/giantswarm/micrologger"
+	"github.com/spf13/afero"
+	yaml "gopkg.in/yaml.v2"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/types"
+	"k8s.io/helm/pkg/helm"
+)
+
+type Config struct {
+	Clients        *Clients
+	GuestFramework *framework.Guest
+	Logger         micrologger.Logger
+
+	AuthToken    string
```

Comment:
What is the `AuthToken` here?

Reply:
It's for the Stormforger API. I'll include that in the name.